### PR TITLE
Add webshop and dashboards for Noordoostpolder

### PR DESCRIPTION
**Pull request by aghimpu**
https://github.com/teamforus/development/issues/215

### CODE REVIEW

**Comment by maxvisser** [comment]
on hold, will update figma prototype

**Comment by dev-rminds** [comment]
@aghimpu Please move this assets (and other assets if any) to implementation resources folder 
image

**Comment by aghimpu** [comment]
> @aghimpu Please move this assets (and other assets if any) to implementation resources folder
> image

Done

**Comment by maxvisser** [comment]
why do we need seperate navbars for noordoostpolder? (pug files)

File: src/sources/forus-platform/js/angular-1/i18n/i18n-nl.js [hunk 1]
```diff
@@ -102,6 +102,10 @@ module.exports = {
                 title: "Meld uw organisatie aan voor het Kerstpakket",
                 subtitle: "Dit is het start scherm om uzelf aan te melden op het aanbieders dashboard.",
             },
+            noordoostpolder: {
+                title: "Meld uw organisatie aan voor de Kindregeling",
```
Comment (maxvisser) [inline src/sources/forus-platform/js/angular-1/i18n/i18n-nl.js:114]: Meld uw organisatie aan voor het Meedoenpakket

File: src/sources/forus-platform/pug/tpl/pages/landing/home-sponsor-noordoostpolder.pug [hunk 2]
```diff
@@ -0,0 +1,31 @@
+landing-navbar(template="navbar-noordoostpolder")
```
Comment (maxvisser) [inline src/sources/forus-platform/pug/tpl/pages/landing/home-sponsor-noordoostpolder.pug]: I think these pages are too similar to home-sponsor.pug Why is this needed?

File: src/sources/forus-webshop/js/angular-1/components/HomeComponent.js [hunk 3]
```diff
@@ -28,12 +28,15 @@ let HomeComponent = function(
         return ModalService.open('modalOpenInMe', {});
     };
 
-    if(AuthService.hasCredentials()) {
-        VoucherService.list().then(res => {
-            $ctrl.vouchers = res.data.data; });
+    $ctrl.$onInit = function() {
```
Comment (maxvisser) [inline src/sources/forus-webshop/js/angular-1/components/HomeComponent.js]: why do we change this on this branch? is this intended?

**Review by maxvisser — CHANGES_REQUESTED** [review]
- [x] remove empty css files image
- [x] add noordoostpolder logo to dashboards
image
- [x] remove seperate pug files for noordoostpolder (footer can stay)
- [x] use as much of common css as possible, there are lot of CSS changes but I think some are not needed.
- [x] resolve  translation comments
- [x] buttons on landing dashboard pages have green border (the pink buttons) should not be;
- [x] login button transitions pink text color to green
- [x] add title (of html index page) "Meedoenpakket webshop"

File: src/sources/forus-webshop/scss/noordoostpolder/includes/shame.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
+// these items have  section as parent class, while other implementatios have block as parent
+
+.section {
+    .section-title {
+        color: #4e4d40;
+        text-align: center;
+        padding-bottom: 1vw;
+    }
+    &.section-profile {
```
Comment (maxvisser) [inline src/sources/forus-webshop/scss/noordoostpolder/includes/shame.scss]: section profile is removed i believe

File: src/sources/forus-webshop/scss/noordoostpolder/sections/blocks/map.scss [hunk 5]
```diff
@@ -0,0 +1,15 @@
+.block {
+    &.block-map {
+        padding: 120px 0 240px;
```
Comment (maxvisser) [inline src/sources/forus-webshop/scss/noordoostpolder/sections/blocks/map.scss]: I think these paddings and margins result in some css bug on frontpage

File: src/sources/forus-webshop/js/angular-1/components/HomeComponent.js (hunk 3 above)
Comment (aghimpu) [reply src/sources/forus-webshop/js/angular-1/components/HomeComponent.js]: just some code refactoring, it wont really affect anything

File: src/sources/forus-platform/pug/tpl/pages/landing/home-sponsor-noordoostpolder.pug (hunk 2 above)
Comment (aghimpu) [reply src/sources/forus-platform/pug/tpl/pages/landing/home-sponsor-noordoostpolder.pug]: I thought there could be more differences at first - on the footer and navbar, but it's using home-sponsor.pug now

File: src/sources/forus-platform/scss/noordoostpolder/landing.scss [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+
+            img {
+                filter: brightness(5);
+            }
+        }
+    }
+}
+
+.header-splash {
+    height: 20.5vw;
```
Comment (maxvisser) [inline src/sources/forus-platform/scss/noordoostpolder/landing.scss:5]: image

**Comment by maxvisser** [comment]
image
profiel is still black

**Comment by maxvisser** [comment]
I would make this all black 
image

**Comment by maxvisser** [comment]
image
not clickable because of some padding / margin css (custom to this implementation?)

**Comment by maxvisser** [comment]
image
steps titles are white

**Comment by maxvisser** [comment]
image
municipality logo is missing; I can send you svg

**Comment by maxvisser** [comment]
image
buttons, carrets and other color definitions on dashboard should stay default blue. no customizations needed.

please revert to standard colors

image

File: src/sources/forus-webshop/js/angular-1/components/HomeComponent.js (hunk 3 above)
Comment (maxvisser) [reply src/sources/forus-webshop/js/angular-1/components/HomeComponent.js]: why i asked: this has merge conflicts with digid alpha branch; so please don't do refactoring on these kinds of branches :)

**Comment by aghimpu** [comment]
> image
> municipality logo is missing; I can send you svg

I think you didn't rerun gulp to copy the new assets, there's a logo there (and the new images on the last commit)

File: src/platforms/Dashboards/DashboardNoordoostpolderProviderPlatform.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
+const platform = require('./_DashboardBasePlatform').clone();
+const destPath = '../dist/forus-platform.provider.noordoostpolder';
+
+// change platform name
+platform.setName('dashboard_noordoostpolder_provider');
+
+// tweaking output and cleaned paths config
+platform.setDest(`${destPath}`);
```
Comment (maxvisser) [inline src/platforms/Dashboards/DashboardNoordoostpolderProviderPlatform.js]: tested this on dev.
Could you please check this file and update them with the 
platform.setDestRootPath(destPath); functionality

Comment (maxvisser) [reply src/platforms/Dashboards/DashboardNoordoostpolderProviderPlatform.js]: same for dashboards

**Comment by maxvisser** [comment]
Could you resolve merge conflicts and fix this issue. https://github.com/teamforus/forus-frontend/pull/828#discussion_r356438609 After this it can be merged. We will make new issues on stakeholder feedback.

**Review by maxvisser — APPROVED** [review]

**Pull request merged**